Java/Varargs/Variable_Length_Arguments/Variable_Length_Arguments.md

```diff
@@ -0,0 +1,119 @@
+# VARIABLE ARGUMENTS (VARAGS)
+       
+In JDK5,Java has included a feature the simplies the code for method that take variable number of arguments, this feature is called **_variable length arguemt (varangs)_**
+  The varargs allows the method to **accept zero or multiple arguments**. Methods which can take variable number of arguemnts is called **_vararg method._**
+  Before varags either we use overloaded method or take an array in the method parameter,but it was not considered good as it lead to large code with more chances of error.
+  So over all this varags is a good and simple option.
```

Comment:
Please fix all the grammatical and spelling mistakes. Use Grammarly or any other platform.